### PR TITLE
doc: regions and zones

### PR DESCRIPTION
**Pull request by DenBeke**

### CODE REVIEW

File: website/docs/regions_and_zones.html.markdown [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## Zones
+
+In order to deploy highly available application, a region can be splitted in many Availability Zones (AZ). Latency between multiple AZ of the same region are low as they have a common network layer.
+
+
+## Resource IDs
+
+To save this notion of regions and zones in the state, all the Terraform IDs of Scaleway contain the region or zone.
+This is saved in the following format: `{zone|region}/{scaleway_id}`. Where `zone` or `region` is the place where the resource is created and where `scaleway_id` is the ID that is used on Scaleway's console/API.
```
Comment (QuentinBrosse) [inline website/docs/regions_and_zones.html.markdown]: `resource_id`

File: website/docs/regions_and_zones.html.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+page_title: "Scaleway Zones and Regions"
+description: |-
+  Scaleway resources can be created in availability zones and regions.
+---
+
+# Scaleway Zones and Regions
+
+Scaleway's products are deployed across multiple datacenter in the world.
+
+For performance and legal reasons, some products are splitted by Region or by Availability Zones. When using such product, you can choose the location that better fits your need (country, latency, ...).
```
Comment (jerome-quere) [inline website/docs/regions_and_zones.html.markdown]: performance => technical

Comment (DenBeke) [reply website/docs/regions_and_zones.html.markdown]: I don't really see the problem with this explanation. It's in Terraform docs. I think we can suppose that there everyone who uses terraform will completely understand this doc, no?

Comment (DenBeke) [reply website/docs/regions_and_zones.html.markdown]: My bad, I misunderstood. Fixed now.

**Pull request merged**